### PR TITLE
VS2022 support

### PR DESCRIPTION
**Pull request by chrpai**
Fix for Issue 6727 : Detect VS2022.  Fully validated.

### CODE REVIEW

**Comment by chrpai** [comment]
A addressed the two concerns.  I don't do this often... you all can squash when you merge right?

File: src/chm/documents/customactions/wixvsextension.html.md [hunk 1]
```diff
@@ -1370,6 +1371,138 @@ Here is a complete list of properties for the <a name="VS2019properties">**Visua
   </tr>
 </table>
 
+Here is a complete list of properties for the <a name="VS2019properties">**Visual Studio 2022**</a> product family:
```
Comment (pomoika) [inline src/chm/documents/customactions/wixvsextension.html.md:1374]:
```suggestion
Here is a complete list of properties for the <a name="VS2022properties">**Visual Studio 2022**</a> product family:
```

Comment (chrpai) [reply src/chm/documents/customactions/wixvsextension.html.md]: Great catch!

**Comment by barnson** [comment]
Closed via https://github.com/wixtoolset/wix3/commit/9871b1061484ea2d8d1a0e4ea08ae6ce5f064c71.

**Pull request closed without merging**